### ISSUE TITLE
Animated number as child of <Text> component does not animate

### ISSUE THREAD

**Issue by fega** [react-spring/react-spring#758]
## 🐛 Bug Report

React spring is not working in react-native

## To Reproduce
```
import React from 'react';
import { useSpring, animated } from 'react-spring/native';
import { Text, View, StyleSheet } from 'react-native';

const s = StyleSheet.create({
  container: {
    alignItems: "center"
  },
  titleText: {
    color: "#fff",
    fontWeight: "bold",
    fontSize: 20
  },
  fadedText: {
    color: "#898989",
    fontSize: 11
  }
});

const AnimatedText = animated(Text)

const StatText = (props) => {
  const value = useSpring({ to: { number: props.value }, from: { number: 100 } })

  return (<View style={s.container}>
    <AnimatedText style={s.titleText}>{value.number}</AnimatedText>
    <Text style={s.fadedText}>{props.name}</Text>
  </View>)
}

export default StatText
```

## Expected behavior

it should update the value but it doesn't happen, or sometimes happen in a buggy way

## Link to repro (highly encouraged)

https://snack.expo.io/@fegamaster/react-spring-react-native-problem

## Environment

- `react-spring` v8.x.x
- `react-native` https://github.com/expo/react-native/archive/sdk-33.0.0.tar.gz
- `react` 16.8.3

**Comment by aleclarson** [comment]
Please try with v9 beta: `npm install react-spring@next`

**Comment by fega** [comment]
@aleclarson Hello, thanks for the quick response, I got the same issue

**Comment by slorber** [comment]
`<AnimatedText style={s.titleText}>{value.number}</AnimatedText>`

This does not make much sense in RN to render the animated value, you have to pass it as style (cf https://github.com/slorber/react-native-animation-hooks/issues/5)

**Comment by fega** [comment]
@slorber thanks so much

**Comment by aleclarson** [comment]
@slorber Wrong, this is supposed to be possible.

**Comment by slorber** [comment]
Oh really? In RN? With React spring only or also animated? How the UI would
animate from one number to the other?

Le jeu. 18 juil. 2019 à 20:02, Alec Larson <notifications@github.com> a
écrit :

**Comment by aleclarson** [comment]
Fixed in `9.0.0-rc.2` (#985)

**Comment by headwinds** [comment]
I'm interested in this solution as well... 

I found a react useSpring version that produces the [desired result](https://codesandbox.io/s/react-spring-interpolate-numbers-2xmgb?file=/src/index.js) - magic right @slorber 

I tried to update @fega example in this [snack](https://snack.expo.io/@headwinds/react-spring-react-native-problem) - it feels close and identical to the react version but I must still be missing something because I only see the starting value.

I struggled to find good snack expo examples when I googled "react-spring expo" so hopefully we can add another working version here along with the awesome codesandbox that @aleclarson created in his fix
